src/components/PageLayout.js

```diff
@@ -169,7 +169,7 @@ export default function Page({
         <Flex
           key={now}
           ref={pageRefCallback}
-          maxW={pageWidthPx}
+          maxW={'var(--docs-page-width)'}
```

Comment:
Can we use the `DOCS_PAGE_WIDTH_VAR` here? I added a suggestion but the constant will also have to be imported at the top of the file.

```suggestion
          maxW={`var(${DOCS_PAGE_WIDTH_VAR})`}
```

Reply:
Good catch! Thanks!